src/configure.py

```diff
@@ -195,9 +197,22 @@ def onmindelegation(input):
             input = "0"
         global parser
         parser.set(MIN_DELEGATION_AMT, float(input))
-        parser.validate_service_fee(parser.get_conf_obj())
```

Comment:
Why are you deleting the check for "validate_service_fee"?

Reply:
Before it felt like a "copy paste" error but now the previous check was removed intentionally?

Reply:
Yes, it's intentional as [`onmindelegation`](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/blob/2c15b08aa15b92769176643f0b27e8ea507655eb/src/configure.py#L194-L204) itself seems to be a copy-paste from [`onservicefee`](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/blob/2c15b08aa15b92769176643f0b27e8ea507655eb/src/configure.py#L139-L150).
Before this change [`onmindelegation`](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/blob/2c15b08aa15b92769176643f0b27e8ea507655eb/src/configure.py#L194-L204) was validated by [`validate_service_fee`](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/blob/2c15b08aa15b92769176643f0b27e8ea507655eb/src/config/yaml_baking_conf_parser.py#L163-L167) which is completely unrelated to `min_delegation_amt` in config but actually validates `service_fee` while proper validation [`validate_min_delegation_amt`](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/blob/2c15b08aa15b92769176643f0b27e8ea507655eb/src/config/yaml_baking_conf_parser.py#L169-L179) for [`onmindelegation`](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/blob/2c15b08aa15b92769176643f0b27e8ea507655eb/src/configure.py#L194-L204) exists but not used at all.

I fixed this issue but if you like I could create a separate MR for this fix but I prefer not to as the size of this change is relatively small.

Reply:
No. perfectly fine. You just see that I am basically reading these parts of the code for the first time :D